igneous/task_creation.py

```diff
@@ -302,6 +298,7 @@ def create_meshing_tasks(
       layer_path,
       mip=vol.mip,
       max_simplification_error=max_simplification_error,
+      obj_id_range=obj_id_range,
```

Comment:
- Why obj_id_range instead of a list of object IDs?

Reply:
a list could be very long, a range is just two numbers. For zebrafish, we only need a range.

Reply:
I don't have any better ideas quite yet, but a shell of an idea that's not JSON serializable so far as I know:

`object_ids=[ 1, 5, 5832, range(20,50) ]`

Reply:
what about... `object_ids=[ 1, 5, 5832, (20, 50) ]`? The nested tuple represents the range.